hw/top_earlgrey/ip/rv_plic/fpv/rv_plic_fpv.core

```diff
@@ -0,0 +1,28 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:fpv:rv_plic_fpv:0.1"
+description: "FPV testbench RISC-V PLIC top_earlgrey"
+
+filesets:
+  files_fpv:
+    depend:
+      - lowrisc:ip:tlul
+      - lowrisc:prim:all
+      - lowrisc:top_earlgrey:rv_plic
+    files:
+      - ../../../../ip/rv_plic/fpv/vip/rv_plic_assert_fpv.sv
```

Comment:
fusesoc does not support '../' in the path. It works flakily. 
Please add core file for the generic fpv files and add that core file as a dependency.

Reply:
oh I remembered, thank you for the reminder Sri. I will update that soon.